avaframe/out3SimpPlot/plotSettings.py

```diff
@@ -0,0 +1,50 @@
+"""
+    Plot settings for output figures
+
+    This file is part of Avaframe.
+"""
+
+import seaborn as sns
+from matplotlib import cm
+import copy
+import matplotlib
+
+# define figure dimentions
+figW = 12
```

Comment:
Can you make this a dictionary so that there is a option to chose 1, 2, 3, 4 column figure and 1, 2, 3, 4 rows?